tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/RunCommandTests.cs

```diff
@@ -80,6 +81,8 @@ public void RedirectInput()
         // dd-env is an argument for the target application and therefore shouldn't set the DD_ENV variable
         var commandLine = $"run -- {startInfo.Executable} {startInfo.Args} echo";
 
+        System.Console.InputEncoding = Encoding.ASCII;
```

Comment:
Out of interest, is it _necessary_ that it's ASCII? I would expect UTF-8 to fix it too (as the problem as I understand it is the _inference_) and there's a _possibility_ we may use UTF-8 chars at some point I guess (obvs we could fix it then if so)